scan-irb/report-en.tex

```diff
@@ -58,7 +58,23 @@
 \begin{description}[font=\normalfont,align=left,labelwidth=12em]
 \item [\textbf{\VAR{pat_name|e}}]
 \item [\textbf{Date of Birth}] \VAR{birth_date|e}
-\item [\textbf{Specimen}] Midnasal, self-collected
+\item [\textbf{Specimen}]
+  %- if swab_type == "mts"
+  Midnasal,
+  %- elif swab_type == "ans"
+  Anterior,
+  %- else
+  \errmessage{Invalid swab_type "\VAR{swab_type}"}
```

Comment:
Do you need to accommodate swab type "none"? See Slack conversation https://seattle-flu-study.slack.com/archives/CCAEWSFTK/p1595442310319000

Reply:
We would filter out swab types of `none`, so we should throw an error if they slip in here.

Reply:
Oh shoot, I conflated this with `unk` in my head. I don't know if we know how to handle `none` right now. We don't have the language around it.

Reply:
See the latest in the Slack thread. It sounds like the "none" swab-type case are samples/results that we won't generate PDFs for.